Common/EventId.h

```diff
@@ -0,0 +1,177 @@
+#pragma once
+
+#include <map>
+
+struct RunLumi {
+  using RunType = unsigned int;
+  using LumiType = unsigned int;
+
+  static constexpr RunType undefinedRun = std::numeric_limits<RunType>::max();
+  static constexpr LumiType undefinedLumi = std::numeric_limits<LumiType>::max();
+
+  static constexpr char separator = ':';
+
+  static const RunLumi& Undefined()
+  {
+    static const RunLumi undefined;
+    return undefined;
+  }
+
+  static std::vector<std::string> Split(const std::string& idStr)
```

Comment:
(just for my knowledge) 
This function and also ToString could be generalised and put in the class CorrectionBase or also in analysis tools. Is it a good idea?

Reply:
we can consider re-using some tools from the previous analysis e.g. https://github.com/hh-italian-group/AnalysisTools/blob/master/Core/include/TextIO.h
But for now I would keep c++ part of the framework as light as possible. Since EventId.h is not used anywhere in the code, I'll remove it from the current PR.